editor/src/components/navigator/left-pane/github-pane/index.tsx

```diff
@@ -2,290 +2,340 @@
 /** @jsx jsx */
```

Comment:
This file should definitely be split up, as it contains multiple components, but that can happen later and shouldn't block this PR